docs/book/v3/migration/v2-to-v3.md

```diff
@@ -251,6 +255,11 @@ Support for these formats has been removed so the following classes are no longe
 
 The following compression formats are still available: `Bz2`, `Gz`, `Tar` and `Zip`
 
+### Removal of the `AbstractSeparator` class
+
+Various filters such as `SeparatorToCamelCase` inherited from the abstract class `AbstractSeparator` to reuse the construction and accessor methods.
+The classes that inherited now use composition to continue provision the expected behaviour, without the risk of changes to the abstract class unintentionally modifying the interface of inheriting classes.
+
```

Comment:
Perhaps just drop the docs changes here - I'll create a TODO issue for the migration guide so that I remember to cover all the relevant topics before 3.0.0 is released 👍

Reply:
Cool. Thanks 
I've taken that out